src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -276,6 +276,27 @@ User preferences are scoped to the current project by default, stored in a local
 
 Using the `--global` flag, user preferences can also be applied to every Astro project on the current machine. Global user preferences are stored in an operating system-specific location.
 
+<h3> Available preferences </h3>
+
+- `devToolbar` — Enable or disable the development toolbar in the browser. (Default: `true`)
+- `checkUpdates` — Enable or disable automatic update checks for the Astro CLI. (Default: `true`)
+
+The `list` command prints the current settings of all configurable user preferences. It also supports a machine-readable `--json` output.
+
+```shell
+astro preferences list
+```
+
+Example terminal output:
+
+| Preference               | Value |
+| ------------------------ | ----- |
+| devToolbar.enabled       | true  <tr></tr>|
```

Comment:
| devToolbar.enabled       | true  |

Reply:
Yeah, I know this looks weird, but this is what's necessary to avoid the zebra striping. Since you don't see that in the terminal output, I was trying to match that visually on the page in Markdown.